{{cookiecutter.project_dirname}}/docker-compose.yaml

```diff
@@ -1,4 +1,4 @@
-version: "3.4"
+version: "3.8"
```

Comment:
```suggestion
version: "3.9"
```

Reply:
In our latest projects, we have tested only version 3.8 so far. For now, I would try version 3.9 in a project generated before reporting it here as well.

Reply:
It was tested here https://github.com/20tab/nextjs-continuous-delivery/blob/main/%7B%7Bcookiecutter.project_dirname%7D%7D/docker-compose.yaml

Reply:
Okay, but it's a different repository. I also agree to update to 3.9, but before doing it here, let's do it and test it on a generated project.